data/json/items/melee.json

```diff
@@ -387,8 +369,7 @@
         "techniques" : ["WBLOCK_1"],
         "volume" : 5,
         "bashing" : 4,
-        "price" : 2400,
-        "price_postapoc": 0
```

Comment:
Don't delete `"price_postapoc" : 0` from useless items. It defaults to `"price"`, not to 0.